goodmap/data_validator.py

```diff
@@ -0,0 +1,61 @@
+import json
+from sys import argv
+
+
+def check_json_validaty(json_file):
+    json_data = json.load(json_file)
+    return json_data
+
+
+def check_obligatory_fields(datapoints, obligatory_fields):
+    for p in datapoints:
+        for field in obligatory_fields:
+            if field not in p.keys():
+                print(f'ERROR, datapoint: \n {p} \n is missing obligatory field: "{field}"')
+
+
+def check_categories_values(datapoints, categories):
+    for p in datapoints:
+        for category in categories & p.keys():
```

Comment:
I meant `&` as a set intersection. Now that you pointed it out, I am a bit surprised that it worked, since `categories` is not a set object but a list. But  `p.keys()` [returns a set-like object](https://docs.python.org/3/library/stdtypes.html#dictionary-view-objects) so apparently if one operand is set-like then it works.
Should I somehow make it more clear in code?

Reply:
Oh. I didn't know that there is intersection operator in python. And its the same as bitwise operator :)
Good job.

For what it's worth - categories probably should be also set, right?

Reply:
Do it on your own dicrection. My comment was just wrong.  If you see this as needed - go for it. If no - close this comment at your own